### DISCUSSION TITLE
MD files hot reloading

### DISCUSSION THREAD

**Discussion by vinicius98s** [vercel/next.js#15581] [Help] [answered]
I started a project following the [blog starter typescript](https://github.com/vercel/next.js/tree/canary/examples/blog-starter-typescript) example and I noticed that there isn't hot reload for the MD files. Is possible to have it with some configuration? It's really annoying have to reload every time I change something.

Any help is appriciated.
Thanks.

**Comment by jamesmosier** [comment, marked as answer]
Fast Refresh only works for files within React's component tree. https://nextjs.org/docs/basic-features/fast-refresh#how-it-works

**Reply by vinicius98s** [reply]
Thanks for a response, I thought a custom webpack config could do this job, but now I guess it's not possible

**Comment by bluprince13** [comment]
I used hashicorp/next-remote-watch to solve this for my blog where I wanted hot reloading for my markdown files.

My commit: bluprince13/bluprince13-next@941c06e
